### PR TITLE
Clean up (tests, travis, tox)

### PR DESCRIPTION
**Pull request by juannyG**
- Python2.6 support
- Move tests out of app
- Migrate to using py.test/py.test-django as the test runner
- Implement tox to start paving the road for testing multiple python/Django/DRF version support
- Travis-CI integration
- Codecov.io integration
- Update README to be accurate with (current) supported versions

@marcosmoyano

### CODE REVIEW

**Comment by juannyG** [comment]
I already know I'm going to need make some tweeks to .travis.yml to run tests through tox, but then run coverage to generate the coverage report that codecov.io needs.

This PR should at least allow anyone to clone the repo, install requirements and run tests successfully with tox.

File: tests/test_routers.py [hunk 1]
```diff
@@ -59,17 +51,22 @@ def test_link_and_action_decorator(self):
         """Test client link & action decorator routing"""
         endpoints = ['action1', 'action2', 'action3', 'link1', 'link2']
         routes = self.router.get_routes(BasicViewSet)
+        import pprint
```
Comment (marcosmoyano) [inline tests/test_routers.py]: please remove this import and prints

File: tests/test_routers.py [hunk 2]
```diff
@@ -111,7 +108,7 @@ def register(self):
         self.router = routers.SimpleRouter()
         self.router.register(r'notes', NoteViewSet)
 
-        from rest_framework_saasy.tests import test_routers
+        from tests import test_routers
```
Comment (marcosmoyano) [inline tests/test_routers.py]: can you move this import to the top level?

**Comment by marcosmoyano** [comment]
:+1:

**Pull request merged**